### PR TITLE
Correctly display "Manage Project Settings" button

### PR DESCRIPTION
**Pull request by EtienneGrall**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation content changes

## Other information

Issue Number: N/A

### CODE REVIEW

File: frontend/src/components/Root/components/Header/Header.tsx [hunk 1]
```diff
@@ -28,17 +29,30 @@ interface Props {
...
+  user: UserState;
 }
 
 export const Header: React.FunctionComponent<Props & InjectedIntlProps> = ({
   currentURL,
+  fetchUserRequest,
   isUserAuthenticated,
   isMenuDisplayed,
   intl,
+  user,
```
Comment (fargito) [inline frontend/src/components/Root/components/Header/Header.tsx:42]: Do you use this ?

**Comment by phacks** [comment]
I’ve tried it on the review app, and it’s working! Thanks @EtienneGrall. I think @fargito is right and the `user` variable isn’t used, could you fix that so that we can merge?

File: frontend/src/components/Root/components/Header/Header.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React, { MouseEvent, useState } from 'react';
 
 import Logo from 'components/Logo';
 import { FormattedMessage, InjectedIntlProps } from 'react-intl';
+import { UserState } from 'redux/user';
```
Comment (fargito) [inline frontend/src/components/Root/components/Header/Header.tsx:5]: Is this used as well ?

File: frontend/src/components/Root/components/Header/Header.wrap.tsx [hunk 3]
```diff
@@ -1,15 +1,19 @@
 import { injectIntl } from 'react-intl';
 import { connect } from 'react-redux';
-import { RootStateWithRouter } from 'redux/types';
-
+import { Dispatch } from 'redux';
 import { getCurrentURL } from 'redux/selectors';
+import { RootStateWithRouter } from 'redux/types';
+import { fetchUserRequest } from 'redux/user';
+import { getUser } from 'redux/user/selectors';
```
Comment (fargito) [inline frontend/src/components/Root/components/Header/Header.wrap.tsx:7]: Is this used too ?

**Comment by phacks** [comment]
Looks good to me!

**Pull request merged**